ci/common-ci.yml

```diff
@@ -76,7 +76,7 @@ variables:
 
 .build_common_base:
   stage: build
-  timeout: 2 hours
+  timeout: 45 minutes
```

Comment:
This is in theory ok, but for GH200 the timeout includes the slurm queuing time, so this could get tight in some situations (GitLab timeout is 45 minutes, and the slurm timelimit is set to 45 minutes based on the GitLab timeout).

I think since this time is adjusted based on the actual build time, I would override this for the GH200 configurations with `SLURM_TIMELIMIT: 00:45:00` and a timeout of e.g. 8 hours or whatever the default GitLab timeout might be.

For the x86 builds that happen on rosa this isn't really a problem since (most of) the time is actually dedicated to the actual build job, modulo runner startup/pulling images etc.

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/1290/commits/5374d09f6dd121d6bdf546ea62c11617baf786d5